scripts/linters/other_linter.py

```diff
@@ -0,0 +1,276 @@
+# coding: utf-8
```

Comment:
The name of this files is not good. Can you please change this? 

Suggestion: Can we keep the original files instead of merging these three checks together and naming it other_linter? We can have a class in each of those original files similar to what you did for codeowner linter.

Reply:
Changed the name to other_file_linter, I did this because all the linter files are named by the file types, for example, python_linter, css_linter, etc. And we have other file types for all the remaining file types, so I think it is better to have this name